modules/sdk-core/src/bitgo/wallet/wallet.ts

```diff
@@ -115,6 +115,12 @@ export class Wallet implements IWallet {
     }
   }
 
+  isEthTss(): boolean {
```

Comment:
Can we do the same as https://github.com/BitGo/BitGoJS/blob/master/modules/sdk-core/src/bitgo/wallet/wallets.ts#L182 until we see the use case expand in being used in multiple places?

Reply:
I am not sure I understand what you're suggesting? 🤔

Reply:
Can we move this logic into a variable without adding a new method in the `build` method the same way we are doing it in `generateWallet`?

Reply:
just pushed a change, let me know if this is what you meant